<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QComboBox" name="BaurdRate">
   <property name="geometry">
    <rect>
     <x>348</x>
     <y>20</y>
     <width>65</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <item>
    <property name="text">
     <string>9600</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>57600</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>115200</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="Connection">
   <property name="geometry">
    <rect>
     <x>161</x>
     <y>20</y>
     <width>181</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QComboBox" name="ComPort">
   <property name="geometry">
    <rect>
     <x>419</x>
     <y>20</y>
     <width>65</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <item>
    <property name="text">
     <string>COM1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>COM2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>COM3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>COM4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>COM5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>COM6</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>COM7</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>COM8</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>COM9</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>COM10</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="Disconnect">
   <property name="geometry">
    <rect>
     <x>561</x>
     <y>20</y>
     <width>65</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Disconnect</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Connect">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>20</y>
     <width>65</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </widget>
  <widget class="QQuickWidget" name="MyquickWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>275</width>
     <height>531</height>
    </rect>
   </property>
   <property name="resizeMode">
    <enum>QQuickWidget::ResizeMode::SizeRootObjectToView</enum>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QQuickWidget</class>
   <extends>QWidget</extends>
   <header location="global">QtQuickWidgets/QQuickWidget</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
